> ## $\color{Orange}\Huge{\textbf{ðŸŽ¯ Goals:}}$  
> 
> * Understand your background and expectations  
> * Gauge your familiarity with computational tools  
> * Introduce basic command-line operations  
> * Establish the course structure, software, and environment (IBIOL server)  

> ### $\color{Orange}\Huge{\textbf{Part 1: Icebreaker}}$    
> #### $\color{Orange}\Huge{\textbf{Expectations and Background}}$     
> * Fill Google [sheet](https://docs.google.com/spreadsheets/d/1QVnIie9xxHmPbUCb4M7Pl1DkW3GBsJpzgN6pkZcXsH4/edit?usp=sharing)  
  
> #### $\color{Orange}\Huge{\textbf{Basic Bioinformatics Concepts}}$    
> *  What is a bioinformatic tool?  
> *  Can you name tools youâ€™ve heard of or used?  
> *  What types of questions can bioinformatics help answer?  
> *  What is FASTA / FASTQ / GFF?   

> ### $\color{Orange}\Huge{\textbf{Part 2: Exercises}}$     
> 
> Designed to assess comfort with UNIX, clusters, and basic file handling.  

> $\color{Orange}\Huge{\textbf{A. Command-Line Essentials}}$  
> 
> *  Log into the cluster / virtual machine (provide access + instructions)  
> *  Navigate to a specified directory (cd, ls, pwd)  
> *  Create your own folder (mkdir)  
> *  Copy a file to your folder (cp)  
> *  Look inside a FASTA file (less, head, cat)  

> $\color{Orange}\Huge{\textbf{B. Data Retrieval and Inspection}}$  
> 
> *  Use wget or curl to download a small dataset from NCBI or ENA  
> *  Uncompress the file (gunzip)  
> *  Count the number of sequences (grep -c '^>')  
> *  Measure file size (du -h)  
